test/omen/default-stakable-tokens-validator/index.test.js

```diff
@@ -0,0 +1,160 @@
+require("../../utils/assertion");
+const BN = require("bn.js");
+const { expect } = require("chai");
+
+const DXTokenRegistry = artifacts.require("DXTokenRegistry");
+const OmenStakableTokenValidator = artifacts.require(
+    "OmenStakableTokenValidator"
+);
+
+contract("OmenStakableTokenValidator", () => {
+    let dxTokenRegistryInstance,
+        omenStakableTokensValidatorInstance,
+        ownerAddress,
+        randomAddress;
+
+    beforeEach(async () => {
+        const accounts = await web3.eth.getAccounts();
+        ownerAddress = accounts[1];
+        randomAddress = accounts[0];
+        dxTokenRegistryInstance = await DXTokenRegistry.new();
+        omenStakableTokensValidatorInstance = await OmenStakableTokenValidator.new(
+            dxTokenRegistryInstance.address,
+            1,
+            { from: ownerAddress }
+        );
+    });
+
+    it("should fail when trying to deploy the contract with a 0-address token registry", async () => {
+        try {
+            await OmenStakableTokenValidator.new(
+                "0x0000000000000000000000000000000000000000",
+                1,
+                { from: ownerAddress }
+            );
```

Comment:
I would add a `throw new Error("should have failed")` here to avoid a false positive.